File: Java/Easy/LengthOfLastWord.java
//Problem's link: https://leetcode.com/problems/length-of-last-word/

//Idea: Start counting when you find a non space char, once you find a space return the count. 
public int lengthOfLastWord(String s) {
	int size = 0;
	final var trimedString = s.trim();
	for(int i=trimedString.length()-1; i>=0; i--) {
		if(trimedString.charAt(i) == ' ') {
			return size;
		} else {
			size++;
		}
	}

	return size;
}

File: Kotlin/Easy/LengthOfLastWord.kt
//Problem's link: https://leetcode.com/problems/length-of-last-word/

//Idea: Start counting when you find a non space char, once you find a space return the count. 
fun lengthOfLastWord(s: String): Int {
    var size = 0
    for(char in s.reversed()) {
        if(char != ' ') {
            size++
        } else if(size != 0) {
            return size
        }
    }
    return size
}